nntrainer/meson.build

```diff
@@ -4,7 +4,7 @@ nntrainer_inc = [
 ]
 
 nntrainer_sources = []
-nntrainer_headers = ['app_context.h']
+nntrainer_headers = ['app_context.h', 'nntrainer_error.h', 'nntrainer_log.h', 'nntrainer_logger.h']
```

Comment:
nntrainer_headers contains the headers which will be installed with nntrainer-devel.
so, if you want to include them, please install them with tizen and ubuntu packaging as well.

btw, why do you need these headers outside nntrainer?

Reply:
Because some of our header file includes those headers making unable to build an application without those header.
We need some cleanup. But for now this patch makes devel working.

Reply:
please update those applications to not use headers. It will be just a small work while making this patch.